common/fileview2/fvidxsource.cpp

```diff
@@ -307,26 +304,11 @@ bool IndexDataSource::getNextRow(MemoryBuffer & out, bool extractRow)
                 }
                 else
                 {
-                    unsigned fileposSize = fileposFieldType->getSize();
-    //              unsigned thisSize = manager->queryRecordSize();             // Should be possible - needs a new function to call cursor->getSize()
                     offset_t filepos;
```

Comment:
This variable name is now a little confusing

Reply:
I guess it's now a dummy - should you have created a version of queryKeyBuffer that does not require it?

Reply:
I think that is the next stage - from investigation this morning it will need to be in the next PR.